src/pages/admin-ui-sdk/extension-points/product.md

```diff
@@ -51,5 +50,6 @@ product: {
 | `confirm.message` | string | No | The message displayed on the confirmation dialog for a mass action |
 | `confirm.title` | string | No | The title of a dialog that confirms the mass action |
 | `label` | string | Yes | An Action label to display in the Mass Actions grid |
-| `path` | string | Yes | The relative path in the application to redirect to the action. The URL will be appended with a query of selected `productIds` |
+| `path` | string | Yes | The relative path in the application to redirect to the action. You might need to prepend `#/` to the path to ensure access to the correct page. The URL will be appended with a query of selected `productIds` |
```

Comment:
We can remove the following: `The URL will be appended with a query of selected productIds`
The product ids are now sent in the `sharedContext`, I guess we need to see how to put an example of the client.